scripts/s3/s3api/bucket_operations.py

```diff
@@ -57,29 +58,30 @@ def __init__(self, access_key: str, secret_key: str, endpoint_url: str, test_id:
         self.object_size = object_size
         self.test_id = test_id
         self.min_duration = 10  # In seconds
-        self.finish_time = datetime.now() + duration \
-            if duration else datetime.now() + timedelta(hours=int(100 * 24))
         self.object_per_iter = 500
         self.iteration = 1
+        self.finish_time = datetime.now() + duration if duration else datetime.now() + timedelta(
+            hours=int(100 * 24))
 
     async def execute_bucket_workload(self):
         """Execute bucket operations workload for specific duration."""
         while True:
-            LOGGER.info("Iteration %s is started...", self.iteration)
+            LOGGER.info("Iteration %s is started for %s...", self.iteration, self.test_id)
             try:
-                if not isinstance(self.object_size, dict):
-                    file_size = self.object_size
-                else:
-                    file_size = random.randrange(self.object_size["start"], self.object_size["end"])
+                file_size = random.randrange(self.object_size["start"], self.object_size["end"])\
+                    if isinstance(self.object_size, dict) else self.object_size
```

Comment:
Why make it one line `if else` if it's not fitting in one line. Takes away all the readability.

Reply:
This is readable and not a complex operations so anyone can read it easily.

Reply:
But I could not understand the reason to change it. IMHO, this is readable but previous code was more readable than current code.

Reply:
Okay let me correct all script. @ketanarlulkar Do you want all conditions to be like this?

Reply:
Just trying to understand the reason. If you think it adds value, you can keep it.